packages/Path-View-Core.package/SClassDiagram.class/class/defaultZoomLevel.st

```diff
@@ -0,0 +1,4 @@
+constants
+defaultZoomLevel
+	
```

Comment:
can you put a method comment which explains the semantics of zoomLevel?
is it actual scale?